src/Components/Postulants/index.js

```diff
@@ -1,11 +1,122 @@
+import { useEffect, useState } from 'react';
 import styles from './postulants.module.css';
+import deleteImage from '../../assets/images/deleteIcon.png';
+import Modal from '../Postulants/Modal';
+
+const Postulants = () => {
+  const [showModal, setShowModal] = useState(false);
+  const [postulants, setPostulants] = useState([]);
+  const [idToDelete, setIdToDelete] = useState('');
+  const [firstNameToDelete, setFirstNameToDelete] = useState('');
+  const [lastNameToDelete, setLastNameToDelete] = useState('');
+
+  useEffect(() => {
+    fetch(`${process.env.REACT_APP_API}/api/postulants`)
+      .then((response) => response.json())
+      .then((response) => setPostulants(response))
+      .catch((error) => console.log(error));
+  }, []);
+
+  const deletePostulant = (id) => {
+    const url = `${process.env.REACT_APP_API}/api/postulants/delete/${id}`;
```

Comment:
If you check on the deployed app in [heroku](https://basd-2021-aq-feature-ms-spwdmd.herokuapp.com/postulants) , the requests to the server are failing cause `/api` is duplicated. It is probably that `/api` already included in the 
`REACT_APP_API` environment variable. You should remove it from here and add it in your `.env` file


![image](https://user-images.githubusercontent.com/8579539/144524841-2ee4a32d-bc95-442e-b5a6-e38df8490594.png)

```suggestion
    const url = `${process.env.REACT_APP_API}/postulants/delete/${id}`;
```

Reply:
Thanks for letting me know where the issue was.